wurst/closures/ClosureForGroups.wurst

```diff
@@ -94,7 +94,10 @@ public function forUnitsInRectCounted(rect r, int count, ForGroupCallback c)
 /** Executes the given closure for every unit in range of the given position */
 public function forUnitsInRange(vec2 pos, real radius, ForGroupCallback c)
 	pushCallback(c)
-	GroupEnumUnitsInRange(DUMMY_GROUP, pos.x, pos.y, radius, filter)
+	GroupEnumUnitsInRange(ENUM_GROUP, pos.x, pos.y, radius + MAX_COLLISION_SIZE, null)
+	for u from ENUM_GROUP
```

Comment:
Why not have a filter for this (should be faster)

Reply:
Because that would be extra global variables to pass to the filter and I am not a fan of that. Not sure if it's faster too.